tools/scripts/android_helpers.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/python3
+
+# -----------------------------------------------------------
+# Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2023-Present Datadog, Inc.
+# -----------------------------------------------------------
+
```

Comment:
Don't we need this file also in the `gitlab` setup for the integration tests in Android ? I see some useful functions here.